grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -77,13 +77,22 @@ class JcrNodesProcessor implements ItemProcessor<JcrNode, Node> {
             final Date afterDate = DateUtil.getDateFromISOString(contentAfterDate)
             log.debug "ContentAfterDate received : ${afterDate}. Will ignore content created or modified before the afterDate"
             final date = getDate(jcrNode)
+            final modifiedDate = getModifiedDate(jcrNode)
             if (!date) {
                 //we want delta content but node doesn't have any date property to compare with. So we ignore it
                 return null
             }
             if (date.before(afterDate)) {
```

Comment:
Working on this now. The way the code is written, it seems like this should be addressed by lines 92-97, where the parent and children are processed together, but I'm testing now to see if that's the case.